tempesta_fw/cache.c

```diff
@@ -293,6 +293,42 @@ tfw_cache_strcpy(char **p, TdbVRec **trec, TfwStr *src, size_t tot_len)
 }
 
 /**
+ * Copies plain or compound (chunked) TfwStr @src to TdbRec @trec may be
+ * appending EOL marker at the end.
+ *
+ * @src is copied (possibly with EOL appended)
+ * @return number of copied bytes on success and negative value otherwise.
+ */
+static long
+tfw_cache_copy_str(char **p, TdbVRec **trec,
```

Comment:
The name is just too close to ```tfw_cache_strcpy()``` and I expect that there will be a lof of confusions about differences between the functions. I'd propose something like ```tfw_cache_streolcpy``` or ```tfw_cache_strcpy_eol```.